packages/image_picker/image_picker_platform_interface/test/new_method_channel_image_picker_test.dart

```diff
@@ -929,6 +929,40 @@ void main() {
         expect(response.file!.path, '/example/path');
       });
 
+      test('getLostData get success response with pathList', () async {
+        picker.channel.setMockMethodCallHandler((MethodCall methodCall) async {
+          return <String, dynamic>{
+            'type': 'image',
+            'path': '/example/path',
+            'pathList': ['/example/path'],
+          };
+        });
+        // ignore: deprecated_member_use_from_same_package
+        final LostDataResponse response = await picker.getLostData();
+        expect(response.type, RetrieveType.image);
+        expect(response.file, isNotNull);
+        expect(response.file!.path, '/example/path');
+        expect(response.files!.first.path, '/example/path');
+      });
+
+      test('getLostData is failed for multiRetrieve method call', () async {
```

Comment:
I'd change this to something a bit more descriptive like:

```suggestion
      test('getLostData returns null for files property on MissingPluginException', () async {
```
(Assuming that's a correct description)

Reply:
I will remove this unit test since the `multiRetrieved` method call was removed. So it is a redundant test I forgot to delete I think.